### DISCUSSION TITLE
Add support for Lagrange multiplier in cfpdes

### DISCUSSION THREAD

**Discussion by Trophime** [feelpp/feelpp#1648] [Toolboxes]
I wonder if it's possible to simply add Lagrange multiplier as unknowns in cfpdes.
If not, It will be great to be able to do that

**Comment by vincentchabannes** [comment]
Currently, it's possible to have a lagrange multiplier on whole domain (as the pressure with Navier-Stokes),  but not on marked faces. Implementation requires some work.
